### PR TITLE
Removed dead code from function commitAllLifeCycles in src/ReactFiber…

### PR DESCRIPTION
**Pull request by bsheikh**
While learning the React Codebase, I believe I've came across some dead code. Let me know if this is being used and I've missed something.

Cheers

### CODE REVIEW

**Comment by react-linter** [comment]
<details>
  <summary>Details of bundled changes.</summary>

  <p>Comparing: 4f459bb144cad7c12e49d8afa2449ff6d9ed9e1f...74ebd7c23f9b31a9cdcfabaada5beba24357f7d0</p>


  
## react-test-renderer
File | Filesize Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip | ENV
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
react-test-renderer.development.js | -0.0% | -0.0% | 375.92 KB | 375.87 KB | 78.72 KB | 78.71 KB | UMD_DEV
**react-test-renderer.production.min.js** | **0.0%** | **-0.3%** | **57.01 KB** | **57.01 KB** | **17.28 KB** | **17.23 KB** | **UMD_PROD**
react-test-renderer.development.js | -0.0% | -0.0% | 366.74 KB | 366.69 KB | 76.05 KB | 76.04 KB | NODE_DEV
**react-test-renderer.production.min.js** | **-0.0%** | **-0.6%** | **56.23 KB** | **56.23 KB** | **16.96 KB** | **16.86 KB** | **NODE_PROD**
ReactTestRenderer-dev.js | -0.0% | -0.0% | 376.04 KB | 375.95 KB | 75.86 KB | 75.84 KB | FB_WWW_DEV

  </details>
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: packages/react-reconciler/src/ReactFiberScheduler.js [hunk 1]
```diff
@@ -679,7 +676,7 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
         }
       } else {
         try {
-          commitAllLifeCycles(root, currentTime, committedExpirationTime);
```
Comment (gaearon) [inline packages/react-reconciler/src/ReactFiberScheduler.js]: Why did you remove `root` here, but `null` above?

Comment (bsheikh) [reply packages/react-reconciler/src/ReactFiberScheduler.js]: Hey, thanks for taking some time to review the PR. As I'm looking back, I'm not sure why I removed null above, root should've been removed instead. Great catch, I've updated the PR.

Thanks!

**Comment by bsheikh** [comment]
Seems like CircleCI was having issues with their docker pull and therefore the tests failed. Is it possible if you could re-run them for me @gaearon? I can't re-run them

**Comment by bsheikh** [comment]
Closing this PR as master now contains the fix :)
